.travis.yml

```diff
@@ -1,8 +1,8 @@
 language: csharp
 mono:
-  - 3.12.0
+  - 5.20.1

```

Comment:
I see 5.x here, and that is mono-4.5 correct?

Reply:
Correct.